hedera-node/configuration/previewnet/application.properties

```diff
@@ -1 +1,4 @@
-
+bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
+contracts.chainId=297
+ledger.id=0x02
+contracts.evm.version.dynamic=true
```

Comment:
We need to learn why this is set in previewnet - it appears only useful in test suites where you can override, for one test, the evm version used.